src/views/settings/view_outfit_editor.cpp

```diff
@@ -0,0 +1,279 @@
+		});
+		ImGui::SameLine();
+
+		components::button("OUTFIT_RANDOM_PROPS"_T, [] {
+			PED::SET_PED_RANDOM_PROPS(self::ped);
+		});
+		ImGui::SameLine();
+
+		components::button("OUTFIT_CLEAR_PROPS"_T, [] {
+			PED::CLEAR_ALL_PED_PROPS(self::ped, 1);
+		});
+		ImGui::SameLine();
+
+		components::button("EXPORT_TO_CLIPBOARD"_T, [] {
+			std::stringstream ss;
+			for (auto& item : components)
+				ss << item.id << " " << item.drawable_id << " " << item.texture_id << " ";
+			for (auto& item : props)
+				ss << item.id << " " << item.drawable_id << " " << item.texture_id << " ";
+			ImGui::SetClipboardText(ss.str().c_str());
+			g_notification_service->push("OUTFIT"_T.data(), "EXPORT_TO_CLIPBOARD"_T.data());
+		});
+		ImGui::SameLine();
+
+		components::button("IMPORT_FROM_CLIPBOARD"_T, [] {
+			std::stringstream ss(ImGui::GetClipboardText());
+			for (auto& item : components)
+			{
+				int id         = 0;
+				int draw_id    = 0;
+				int texture_id = 0;
+				ss >> id;
+				ss >> draw_id;
+				ss >> texture_id;
+				PED::SET_PED_COMPONENT_VARIATION(self::ped, id, draw_id, texture_id, PED::GET_PED_PALETTE_VARIATION(self::ped, id));
+			}
+			for (auto& item : props)
+			{
+				int id         = 0;
+				int draw_id    = 0;
+				int texture_id = 0;
+				ss >> id;
+				ss >> draw_id;
+				ss >> texture_id;
+				if (draw_id == -1)
+					PED::CLEAR_PED_PROP(self::ped, id, 1);
+				else
+					PED::SET_PED_PROP_INDEX(self::ped, id, draw_id, texture_id, TRUE, 1);
+			}
+		});
+
+		ImGui::BeginGroup();
+		for (auto& item : components)
+		{
+			ImGui::SetNextItemWidth(60);
+			if (ImGui::InputInt((item.label + " [0," + std::to_string(item.drawable_id_max) + "]").c_str(), &item.drawable_id, ImGuiInputTextFlags_None))
+				g_fiber_pool->queue_job([item] {
+					PED::SET_PED_COMPONENT_VARIATION(self::ped, item.id, item.drawable_id, 0, PED::GET_PED_PALETTE_VARIATION(self::ped, item.id));
+				});
+		}
+		ImGui::EndGroup();
+
+		ImGui::SameLine();
+
+		ImGui::BeginGroup();
+		for (auto& item : components)
+		{
+			ImGui::SetNextItemWidth(60);
+			if (ImGui::InputInt(
+			        (item.label + " OUTFIT_TEX"_T.data() + +" [0," + std::to_string(item.texture_id_max) + "]").c_str(),
+			        &item.texture_id,
+			        ImGuiInputTextFlags_None))
+				g_fiber_pool->queue_job([item] {
+					PED::SET_PED_COMPONENT_VARIATION(self::ped, item.id, item.drawable_id, item.texture_id, PED::GET_PED_PALETTE_VARIATION(self::ped, item.id));
+				});
+		}
+		ImGui::EndGroup();
+
+		ImGui::SameLine();
+
+		ImGui::BeginGroup();
+		for (auto& item : props)
+		{
+			ImGui::SetNextItemWidth(60);
+			if (ImGui::InputInt((item.label + " [0," + std::to_string(item.drawable_id_max) + "]").c_str(), &item.drawable_id, ImGuiInputTextFlags_None))
+				g_fiber_pool->queue_job([item] {
+					if (item.drawable_id == -1)
+						PED::CLEAR_PED_PROP(self::ped, item.id, 1);
+					else
+						PED::SET_PED_PROP_INDEX(self::ped, item.id, item.drawable_id, 0, TRUE, 1);
+				});
```

Comment:
Wrap this in curly brackets, this is a multiline if statement making it harder to read.